backend/logic/users/users.js

```diff
@@ -99,8 +112,15 @@ var insertAdminUser = async (username, password, email, id) => {
   let hash = hasher.hashPass(password);
   try {
     await connection.beginTransaction();
-    await connection.query("INSERT INTO users VALUES(?, ?, ?);", [username, email, hash]);
-    await connection.query("INSERT INTO staff_members VALUES(?, ?);", [id, username]);
+    await connection.query("INSERT INTO users VALUES(?, ?, ?);", [
```

Comment:
i know this isnt you but can you fix it here too please? 
```suggestion
    await connection.query("INSERT INTO users (username, email, password) VALUES(?, ?, ?);", [
```